static/app/views/organizationGroupDetails/groupEvents.spec.jsx

```diff
@@ -230,6 +241,59 @@ describe('groupEvents', function () {
       );
     });
 
+    it('does not display attachments column with no attachments', async () => {
+      render(
+        <GroupEvents
+          organization={org.organization}
+          api={new MockApiClient()}
+          params={{orgId: 'orgId', projectId: 'projectId', groupId: '1'}}
+          group={group}
+          location={{query: {environment: ['prod', 'staging']}}}
+        />,
+        {context: routerContext, organization}
+      );
+      await waitForElementToBeRemoved(document.querySelector('div.loading-indicator'));
+      const attachmentsColumn = screen.queryByText('attachments');
```

Comment:
Does the table have ARIA roles? This would be better as `queryByRole(`tablecell`, {...etc` if that's possible, since "Attachments" could appear anywhere on the screen for any number of reasons